dev/scripts/src/alluxio.org/build-distribution/cmd/flags.go

```diff
@@ -73,24 +73,19 @@ func handleUfsModulesAndLibJars() error {
 	switch strings.ToLower(includedLibJarsFlag) {
 	case libJarsAll:
 		var allLibJars []string
-		for jar := range coreLibJars {
-			allLibJars = append(allLibJars, jar)
-		}
-		for jar := range additionalLibJars {
+		for jar := range libJars {
 			allLibJars = append(allLibJars, jar)
 		}
 		includedLibJarsFlag = strings.Join(allLibJars, ",")
-	case libJarsCore:
+	case libJarsFuse:
 		var coreJars []string
```

Comment:
```suggestion
		var fuseJars []string
```
and rename for lines below